manifests/init.pp

```diff
@@ -216,14 +216,16 @@
 
   Hash $grafana_config,
 
+  Optional[String[1]] $archive_source = undef,
```

Comment:
Do we want to allow any string or just URLs like so?

```suggestion
  Optional[Stdlib::HTTPUrl] $archive_source = undef,
```

Reply:
It can also be a local share or any http/ftp url.